odl/discr/lp_discr.py

```diff
@@ -258,6 +258,97 @@ def element(self, inp=None):
                                   '{!r}: {}'.format(self, inp, err)), err)
 
     @property
+    def examples(self):
+        """Return example functions in the space.
+
+        Example functions include:
+
+        Zero
+        One
+        Heaviside function
+        Hypercube
+        Hypersphere
+        Gaussian
+        Gradients
```

Comment:
Now that I think of it, shouldnt these be examples in the underlying uspace, and we simply discretize these here? That way we could move this to the general discretization class.